Vagrantfile

```diff
@@ -52,16 +52,20 @@ def vm_cpus
 end
 
 Vagrant.configure("2") do |config|
-  # always use Vagrants insecure key
+  # always use Vagrant's insecure key
   config.ssh.insert_key = false
-
   config.vm.box = "coreos-%s" % $update_channel
-  config.vm.box_version = ">= 681.2.0"
-  config.vm.box_url = "http://%s.release.core-os.net/amd64-usr/current/coreos_production_vagrant.json" % $update_channel
+  config.vm.box_version = "= 647.2.0"
```

Comment:
> it didn't work for me, and shouldn't work

I was surprised it did. Not sure why.

The real issue with breaking the update channel is it kills our nightly "canary" jobs in CI. It is useful to see when breaking changes are introduced into coreos-beta and coreos-alpha so we can remedy things before they get to stable. Without the vagrant cloud update channels working, I don't know how we can build and test against the latest versions.